src/style.css

```diff
@@ -1,3 +1,65 @@
-body {
+:root {
+  --blue-color: #00acdc; /* Определение переменной для синего цвета */
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.nav_list {
```

Comment:
The class name `.nav_list` should be written as `.nav__list` to represent an element of the `nav` block following the BEM (Block Element Modifier) naming convention.